content/applications/hr/recruitment/recruitment-flow.rst

```diff
@@ -0,0 +1,189 @@
+   the applicant's card.
+
+   To change the stage on the applicant's card, click on the applicant's card in the Kanban view to
+   go to a detailed view of the card. The current stage for the card is highlighted at the top,
+   above the card. Click the desired stage for the card, and the stage changes. A log note
+   indicating the stage change appears in the chatter. Navigate back to the Kanban view by clicking
+   on :guilabel:`Applications` in the breadcrumb menu in the top-left, and the applicant's card now
+   appears in the new stage.
+
+  .. image:: recruitment-flow/stage-change.png
+     :align: center
+     :alt: Change the stage of an applicant by clicking on the desired stage at the top of the
+           applicant's card.
+
+.. _recruitment/first-interview:
+
+First interview
+===============
+
+After an applicant has passed the :guilabel:`Initial Qualification` stage, they can be moved to the
+:guilabel:`First Interview` stage in the job position Kanban view. To move the applicant to the next
+stage, drag and drop the applicant's card to the :guilabel:`First Interview` stage, or click on the
+:guilabel:`First Interview` stage button at the top of the individual applicant's card.
+
+.. image:: recruitment-flow/move.png
+   :align: center
+   :alt: An applicant's card moves from one stage to another by using the click and drag method.
+
+.. tip::
+   The :guilabel:`First Interview` stage can be modified so when the applicant's card moves to the
+   :guilabel:`First Interview` stage, an email is automatically sent to the applicant stating an
+   interview is requested. In this pre-configured email template, a link to the recruiting team's
+   calendar appears, allowing the applicant to schedule their interview.
+
+   :ref:`Edit <recruitment/edit-stage>` the :guilabel:`First Interview` stage, and select the
+   :guilabel:`Recruitment: Schedule Interview` for the :guilabel:`Email Template` field, to automate
+   this action.
+
+.. _recruitment/second-interview:
+
+Second interview
+================
+
+After an applicant has passed the :guilabel:`First Interview` stage, they can be moved to the
+:guilabel:`Second Interview` stage. To move the applicant to the next stage, drag and drop the
+applicant's card to the :guilabel:`Second Interview` stage, or click on the :guilabel:`Second
+Interview` stage button at the top of the individual applicant's card.
+
+When the applicant's card moves to the :guilabel:`Second Interview` stage, there are no automatic
+activities or emails configured for this stage. The recruiter can now :ref:`schedule a second
+interview <recruitment/schedule_interviews/recruitment-scheduled>` with the applicant, following the
+same process as the first interview.
+
+.. _recruitment/contract-proposal:
+
+Contract Proposal
+=================
+
+After the applicant has completed the various interview processes, the next step is to send the
```

Comment:
Do you think it would be beneficial to mention moving it to the Contract Proposal stage (in the same way you've described moving it to other stages, up to this point)? Also, is there any benefit to discussing the folded 'Contract Signed' stage? I think if those were added, it could fully round out the entire flow you are describing. What do you think?

Reply:
I was thinking about that but I do have that step included in that document, and always worry about the doubling of information. I will add that though, just so this is thorough!